build.gradle

```diff
@@ -63,6 +63,7 @@ subprojects {
             exclude group: 'org.hamcrest', module: 'hamcrest-core'
         }
         testCompile 'org.hamcrest:hamcrest-all:1.3'
+        compile 'com.google.code.findbugs:annotations:3.0.1'
```

Comment:
you should probably switch your jdk to 6 so that you can run this stuff with the older version of findbugs which is compatible with our target platform.